src/components/site/Breadcrumbs.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Breadcrumbs = props => {
+  return (
+    <ul className="dg_breadcrumbs">
+      <li>
+        <a href="#">
```

Comment:
This should be "/"

Reply:
@martypowell if you mean link titles, I don't think we need those here.

Reply:
I think we need to include a span here, with the class `visuallyhidden` with the text "Home"

Reply:
Updated href to / and added a title to the a tag. Should be reflected in the latest commit.

Reply:
or as @danfox01 suggests follow this.

https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility

Reply:
Tentatively committed. Testing locally some more before giving the "ready for review".

Reply:
Committed and ready for review.